inst/include/lib/NiftiImage.h

```diff
@@ -1266,6 +1314,11 @@ inline NiftiImage & NiftiImage::reorient (const int icode, const int jcode, cons
     return *this;
 }
 
+
+
+
+
+/*
```

Comment:
This commented-out function (which you flag up in your commit message) is the problem, since it's used by my R interface functions. I don't see why it should cause problems in a non-R context, so I guess it can just be uncommented?